backend/handlers/users/usercredit.go

```diff
@@ -0,0 +1,64 @@
+package usershandlers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"socialpredict/setup"
+	"socialpredict/util"
+
+	"github.com/gorilla/mux"
+	"gorm.io/gorm"
+)
+
+// appConfig holds the loaded application configuration accessible within the package
+var appConfig *setup.EconomicConfig
+
+func init() {
+	var err error
+	appConfig, err = setup.LoadEconomicsConfig()
+	if err != nil {
+		log.Fatalf("Failed to load configuration: %v", err)
+	}
+}
+
+type UserCreditResponse struct {
+	Credit int `json:"credit"`
+}
+
+// for usage on sidebar or continuously throughout application in order to continuously show available spend
```

Comment:
// &lt;function name&gt; gets the users available credits for display

(I'll add this to our style guide as well)